src/features/Instructors/AddInstructors/index.jsx

```diff
@@ -1,4 +1,4 @@
-// This component will be modified according to the new wirefrime
+// This component will be modified according to the new wire frame
```

Comment:
![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/b4a94d23-5a89-4e2a-84c1-2161b4e56dea)

@01001110J

Reply:
This line is no longer needed, please remove it